challenge5.js

```diff
@@ -0,0 +1,35 @@
+// const readline = require('readline')
+const rl = readline.createInterface({
+    input:process.stdin,
+    output:process.stdout
+});
+
+function getNumberArray() {
+    return new Promise(resolve => {
+        rl.question('Please enter a list of numbers separated by spaces: ', answer => {
+            const numbers = answer.split(' ').map(Number);
+            resolve(numbers);
+        });
+    });
+}
+
+async function sortNumber() {
+    const numbers = await getNumberArray();
+    let swapped;
+    do {
+      swapped = false;
```

Comment:
Since this section only runs once, why haven't you considered assigning the `false` value when declaring the variable?

Reply:
I did not assign false when declaring the swapped variable because it will be reassigned later in the loop however I can assign false to the swapped variable when declaring it .